VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -117,7 +121,7 @@ String getUserEnrolledCoursesURL(@NonNull String username) {
             try {
                 json = IOUtils.toString(input.in(), Charset.defaultCharset());
             } catch (IOException e) {
-                throw new RetroHttpException(e);
+                throw new RetroHttpException(RetrofitError.networkError(null, e));
```

Comment:
The is no `Response` object if we're getting the JSON data from the custom cache. We should probably be switching to OkHttp's cache anyway when we switch the courseware module to Retrofit, so this approach can just be considered a hack for now.